src/components/paragraphs/stanford-card/card-paragraph.tsx

```diff
@@ -64,12 +62,12 @@ const CardParagraph = ({paragraph, ...props}: Props) => {
         </div>
       }
 
-      <Wysiwyg html={paragraph.suCardBody?.processed} className="order-3"/>
+      <Wysiwyg html={paragraph.suCardBody?.processed} className="order-3 *:text-21 rs-pb-2"/>
 
       {paragraph.suCardLink?.url &&
         <div className="order-4">
           {behaviors.su_card_styles?.link_style === "action" &&
-            <ActionLink href={paragraph.suCardLink.url}>
+            <ActionLink href={paragraph.suCardLink.url} className="font-normal no-underline hocus:underline">
```

Comment:
Should that be the standard style for action link?

Reply:
The standard style hover/focus state for the action link is a bit different. I might have to rework this PR a bit once #9 has been merged in since I make some overlapping adjustments [here](https://github.com/SU-SWS/press-nextjs/pull/9/files#diff-4ad4d8f64d8ae9698c978ee3a276591142691df26c068d4081cd70dee833c35cR15). 🤔 

<img width="976" alt="image" src="https://github.com/SU-SWS/press-nextjs/assets/34019925/5c99a80a-21a9-4fc1-918e-e3330c970d40">